internal/modules/module.go

```diff
@@ -121,7 +122,10 @@ func (m *Module) RegisterExtensions(ctx context.Context, log logrus.FieldLogger,
 		return nil
 	}
 
-	return ext.RegisterExtension(ctx, m.URI, m.Name, m.Version)
+	version := &resolver.Version{
```

Comment:
I'd suggest using the function provided by gobox instead, it's meant to handle cases like this: https://pkg.go.dev/github.com/getoutreach/gobox/pkg/cli/updater/resolver#NewVersionFromVersionString

Reply:
Oh right, I thought that this might be the psudeo-version string but I guess it's not nvm!